packages/test/test-version-utils/src/test/versionUtils.spec.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { getRequestedRange } from "../versionUtils";
+
+describe("Get the major version number above or below the baseVersion", () => {
```

Comment:
What about: minor bumps, patch bumps, malformed versions etc? Could you please add test cases for those.